packages/expo-updates/ios/EXUpdates/UpdatesStateMachine.swift

```diff
@@ -160,8 +160,13 @@ internal struct UpdatesStateContext {
   let downloadedManifest: [String: Any]?
   let checkError: [String: String]?
   let downloadError: [String: String]?
+  let lastCheckForUpdateTime: Date?
 
   var json: [String: Any?] {
+    var dateString = self.lastCheckForUpdateTime?.description
+    if #available(iOS 15.0, *) {
```

Comment:
How do we do date serialization over the bridge in other modules? Or is this the first instance of needing this (grep indicates it might be)?

Reply:
(the reason I ask is that doing `description` usually produces inconsistent results as it is more commonly for debugging)

Reply:
Yes other dates are coming from the server, not generated on the client... I'll have a look to see what we do across the monorepo.

Reply:
An alternative is to do what we already have in `UpdatesLogger`, where we convert to milliseconds since 1/1/70 and record that as a number. That gets put into the `timestamp` field of the updates logs.

Reply:
I like ISO8601 in general. Is there a way we could generate it using `NSISO8601DateFormatter` for older platforms?

Reply:
I'll try that. Is there a similar API for Android?

Reply:
IIRC something like DateTimeFormatter

Reply:
Ok both platforms generate ISO8601 strings now.